src/App.svelte

```diff
@@ -1,372 +1,26 @@
 <script>
-	// buttons are so 2000's, let's use emojis instead
-	import Emoji from './Emoji.svelte';
+  import Footer from "./Footer.svelte";
+  import Header from "./Header.svelte";
+  import TaskList from "./TaskList.svelte";
 
-	// tooltips are helpful
-	import ToolTip from './ToolTip.svelte';
-
-	// string task to do
-	let input = '';
-
-	// array of tasks to do
-	let tasks = [];
-
-	// array of active tasks
-	let activeTasks = [];
-
-	// flag to handle alert status
-	let alertsOn = true;
-
-	// array of completed tasks
-	let completedTasks = [];
-
-	// allows us to track the input field
-	let inputField;
-
-	// toggles mode
-	let mode = "all";
```

Comment:
`tasks`, `mode`, and `count` got moved to the `stores.js` file. Is using a Svelte store overkill for this app? Yeah, kind of. But we'll be using them in FF8 so it's nice to learn about them now.

Reply:
Makes sense to me